src/PolykeyClient.ts

```diff
@@ -124,9 +156,12 @@ class PolykeyClient {
 
   public async destroy() {
     this.logger.info(`Destroying ${this.constructor.name}`);
+    await this.ws.destroy({ force: true });
     await this.session.destroy();
     this.logger.info(`Destroyed ${this.constructor.name}`);
   }
 }
```

Comment:
Try not to hard code force here. If it's an option, it should be an option on the `PolykeyClient.destroy` as well.

```suggestion
  public async destroy({ force = true }: { force?: boolean }) {
    this.logger.info(`Destroying ${this.constructor.name}`);
    await this.ws.destroy({ force });
    await this.session.destroy();
    this.logger.info(`Destroyed ${this.constructor.name}`);
  }
}
```

Reply:
We should discuss whether `force` should be defaulted to true here... since we never do it anywhere else except for transport. So we should understand what the effects here.